pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -44,8 +44,10 @@ export class FileAutoComplete extends React.Component {
         this.onCreateOption = this.onCreateOption.bind(this);
 
         this.debouncedChange = debounce(300, (value) => {
-            if (!value)
+            if (!value) {
+                this.clearSelection();
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18262-20230130-103616-c4037d00-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)